MapboxNavigation/InstructionsBannerView.swift

```diff
@@ -92,4 +92,18 @@ open class BaseInstructionsBannerView: UIControl {
         
         distance = 100
     }
+    
+    /**
+     Updates the instructions banner for a given `RouteProgress`.
+     */
+    public func update(for routeProgress: RouteProgress) {
```

Comment:
Looks like `routeProgress` is used only for its `currentLegProgress`, so make this method take a RouteLegProgress instead.